### PR TITLE
Adding a sample CMS file

### LINKED ISSUE #131: Create small CMS root test file

### PR DESCRIPTION
**Pull request by gordonwatts**
Adds a CMS file in the base cms test directory

Fixes #131

### CODE REVIEW

**Review by sudo-panda — APPROVED** [review]
It looks good to me

**Comment by sudo-panda** [comment]
This also fixes #136

File: func_adl_xAOD/template/cms/r5/analyzer_cfg.py [hunk 1]
```diff
@@ -6,10 +6,16 @@
 
 process.maxEvents = cms.untracked.PSet(input=cms.untracked.int32(-1))
 
+fileNamesArray = []
+with open('/scripts/filelist.txt', 'r') as flist:
+    for line in flist.readlines():
```
Comment (gordonwatts) [inline func_adl_xAOD/template/cms/r5/analyzer_cfg.py:11]: Should this just be list-comprehension. Something like:

```
fileNamesArray = [f'file:{line}' for line in flist.readlines()]
```
```

**Review by gordonwatts — COMMENTED** [review]
Fantastic. Just take a look at my one comment and then I think it is ok for you to merge this puppy! Nice!!!!

File: func_adl_xAOD/template/cms/r5/analyzer_cfg.py (hunk 1 above)
Comment (sudo-panda) [reply func_adl_xAOD/template/cms/r5/analyzer_cfg.py:11]: @gordonwatts would this be better:
```
filelistPath = '/scripts/filelist.txt'
fileNamesArray = [f'file:{line}' for line in open(filelistPath, 'r').readlines()]
fileNamesTuple = tuple(fileNamesArray)
```

Comment (gordonwatts) [reply func_adl_xAOD/template/cms/r5/analyzer_cfg.py:11]: Or:
```
filelistPath = '/scripts/filelist.txt'
fileNamesTuple = tuple(f'file:{line}' for line in open(filelistPath, 'r').readlines())
```

But now we are down in the weeds. :-)

Comment (gordonwatts) [reply func_adl_xAOD/template/cms/r5/analyzer_cfg.py:11]: Any of them will do - make the change and then merge, I think!

Comment (sudo-panda) [reply func_adl_xAOD/template/cms/r5/analyzer_cfg.py]: I also added a shebang to remember that the analyzer uses python instead of python3.

**Pull request merged**